src/System.Collections.Concurrent/tests/ProducerConsumerCollectionTests.cs

```diff
@@ -731,25 +731,19 @@ public void ManyConcurrentAddsTakes_EnsureTrackedCountsMatchResultingCollection(
         }
 
         [Theory]
-        [InlineData(ConcurrencyTestSeconds)]
-        public void ManyConcurrentAddsTakes_CollectionRemainsConsistent(double seconds)
+        [InlineData(1000)]
```

Comment:
This needs to be much bigger.  The purpose of the test is increase the chance of concurrency, hence why it was running for some number of seconds.  It'll complete 1000 items so quickly, there likely won't be any concurrency.